config/config.sample.yaml

```diff
@@ -1,17 +1,40 @@
-# Port for incoming slack requests from webhooks and event API messages.
-slack_hook_port: 9898
-# Prefix of incoming requests to strip when handing. This is NOT the bind host
-  #NOTE: Unlike most of the other urls, this one cannot use localhost,
-  #as this one must be publicly visible to the slack API
-inbound_uri_prefix: "https://my.server.here:9898/"
+homeserver: # Required
+  # The domain name of your homeserver
+  server_name: my.server.here
+  # The URL which the bridge server can reach the homeserver on.
+  # You can use localhost so long as the bridge and homeserver are
+  # hosted on the same machine
+  # 8008 is assumed to be the port the homeserver listens on
+  url: http://localhost:8008
+  # The public facing url for media on your homeserver.
+  # This is usually the public url of your homeserver.
+  # Optional, will use `url` by default.
+  # media_url: "http://my.server.here"
+  # Optional. The maximum size of a uploaded file to Matrix in bytes. No limit by default
+  # max_upload_size: 104857600
+  # Optional. Used to specify the port of the appservice in the config, rather than the cli.
+  # If this is defined, it will **override** the port given in the process arguments.
+  # appservice_port: 9999
 
-# Both of the above options are optional if using RTM (see below), required otherwise.
+# Settings for database connection
+db:
+  # Which database engine to use
+  engine: "postgres" # "postgres" or "nedb". If using "nedb", do not define connectionString
```

Comment:
Will `connectionString` be unused or does the user have to comment out `connectionString` if using nedb?